### DISCUSSION TITLE
wheather clientAppSetup.{js,ts} can be supported in .vuepress

### DISCUSSION THREAD

**Discussion by slgluo** [vuepress/core#268] [Q&A] [answered]
I put `clientAppSetup.ts` and `clientAppEnhance.ts` in `.vuepress`，`clientAppEnhance` be call but not `clientAppSetup`

**Comment by yzx9** [comment]
Please provide a reproduce

**Reply by slgluo** [reply]
[here is my project](https://github.com/slgluo/autoplay.git)

**Comment by Mister-Hope** [comment, marked as answer]
The project link is broken, and the 2 files definily works in latest version, so please check your own codes.
